projects/i18n/switch/language-chunk-loader-token.ts

```diff
@@ -0,0 +1,10 @@
+import {InjectionToken} from '@angular/core';
+
+import {TuiLangChunkLoader} from './options';
+
+export const TUI_LANGUAGE_CHUNK_LOADER = new InjectionToken<TuiLangChunkLoader>(
+    `Webpack chunk loader for Taiga UI libraries i18n`,
+    {
+        factory: () => null,
```

Comment:
Not sure it makes sense to add `null` to `loader` type. Maybe instead you can drop default factory and optionally inject it?

Reply:
fixed